scripts/SANS/isis_reduction_steps.py

```diff
@@ -4282,7 +4282,7 @@ def execute(self, reducer, workspace):
             Reads the geometry information stored in the workspace
             but doesn't replace values that have been previously set
         """
-        _reducer = reducer  # noqa
+        _ = reducer  # noqa
```

Comment:
I think I'm missing something. What's the reason that this line needs to exist at all?